apps/explorer/src/components/common/Card/CardContent.tsx

```diff
@@ -79,10 +79,20 @@ const CardBody = styled.div<{
         font-size: 11px;
         margin-left: ${({ variant }): string => (variant === '2row' ? '0.5rem' : '0')};
         margin-top: ${({ variant }): string => (variant === '2row' ? '0' : '6px')};
-        color: ${({ theme, captionColor }): string => captionColor && theme[captionColor]};
+        color: ${({ captionColor }): string => {
```

Comment:
Nitpick: the code is duplicated bellow, should be moved to a function

Reply:
Addressed